HISTORY.md

```diff
@@ -38,6 +38,9 @@
 * Add support for timestamped snapshots (#9879)
 * Provide support for AbortIO in posix to cancel submitted asynchronous requests using io_uring.
 * Add support for rate-limiting batched `MultiGet()` APIs
+* Add DB properties "rocksdb.blob-cache-capacity", "rocksdb.blob-cache-usage", "rocksdb.blob-cache-pinned-usage" to show blob cache usage.
```

Comment:
Nit: since all of the three items here fall into the same category, maybe use a two-level structure as the following
```
* Expose  perf context statistics, and DB properties for blob cache
  * first item,
  * second item,
  * third item.
```

Reply:
Also, this PR sits on the borderline of "new feature" and "API change", or both